stack.yaml

```diff
@@ -1,11 +1,13 @@
-resolver: lts-11.13
+resolver: lts-12.7
 
 flags:
   ether:
     disable-tup-instances: true
 
 extra-package-dbs: []
 
+allow-newer: true
```

Comment:
This needs to go.

@avieth I think you are the only one(?) who has access to https://github.com/input-output-hk/engine.io, could you give access to the `cardano-sl` group? The engine-io-wai module needs a version bump; and we might want to merge in the version bumps from upstream as well.